js/data/locale/af/NA/dateformats.json

```diff
@@ -3,46 +3,40 @@
         "order": "{date} {time}",
         "date": {
             "dmwy": {
-                "f": "EEEE dd MMMM yyyy",
-                "l": "EEE dd MMMM yyyy",
+                "s": "E yyyy-MM-dd",
                 "m": "EE dd MMM yyyy",
-                "s": "E yyyy-MM-dd"
+                "l": "EEE dd MMMM yyyy",
```

Comment:
The reason to have many diff in `dateformats.json` is just  changed the data format order.. by changed "to s" come first

Reply:
true, we should sort them so that they always come out in the same order

Reply:
or at least always after this

Reply:
yes. I re-generated dateformats.json and sysres.json files by standalone running gendatefmt2.js file. It saves files changes from 1830 -> 1537 in this PR.
As you said we should sort them to get same order all the time.

Reply:
I just implemented the sorting but it makes such a huge change that it makes it impossible to see the differences. So, I put in the treeSort branch and when this PR is done, I'll make a new PR from that branch that contains the exact same settings, but sorted. That way, we know nothing is changed except the order.

Reply:
oh I see. That's good : )